kong/db/dao/plugins.lua

```diff
@@ -126,31 +126,76 @@ local function implements(plugin, method)
 end
 
 
-local function load_plugin_handler(plugin)
-  -- NOTE: no version _G.kong (nor PDK) in plugins main chunk
+local load_plugin_handler do
 
-  local plugin_handler = "kong.plugins." .. plugin .. ".handler"
-  local ok, handler = utils.load_module_if_exists(plugin_handler)
-  if not ok then
-    ok, handler = plugin_servers.load_plugin(plugin)
-    if type(handler) == "table" then
-      handler._go = true
+  local function valid_priority(prio)
+    if type(prio) ~= "number" or
+       prio == (0/0) or  -- NaN
```

Comment:
```suggestion
       prio ~= prio or  -- NaN
```

Reply:
Same thing no? No strong opinions. Both work. If this is all then please merge the suggestion and then the PR.

Reply:
not same, correction. i don’t think both works.